### PR TITLE
Merge GeoBrowser to production

### PR DESCRIPTION
**Pull request by emanueldima**
PR to track moving the GeoBrowser to production. We must test it first with various/random files.

### CODE REVIEW

File: tools/DARIAH-DE-Geo-Browser-CSV.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
```
Comment (andmor-) [inline tools/DARIAH-DE-Geo-Browser-CSV.json:3]: Is the intention to keep it hidden in production?

File: tools/DARIAH-DE-Geo-Browser-KML.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
```
Comment (andmor-) [inline tools/DARIAH-DE-Geo-Browser-KML.json:3]: Same here

File: tools/DARIAH-DE-Geo-Browser-CSV.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        "none"
+    ],
+    "mimetypes": [
+        "text/csv",
+        "text/comma-separated-value",
+        "application/vnd.dariahde.geobrowser.csv"
+    ],
+    "languages": [],
+    "langEncoding": "639-1",
+    "url": "https://geobrowser.de.dariah.eu/beta/",
```
Comment (andmor-) [inline tools/DARIAH-DE-Geo-Browser-CSV.json:27]: Are we going to link to a beta to our production instance? Do we have an idea about the maturity and stability of the beta?

File: tools/DARIAH-DE-Geo-Browser-KML.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        "none"
+    ],
+    "mimetypes": [
+        "application/vnd.google-earth.kml+xml",
+        "application/vnd.google-earth.kmz",
+        "application/vnd.dariahde.geobrowser.kml+xml"
+    ],
+    "languages": [],
+    "langEncoding": "639-1",
+    "url": "https://geobrowser.de.dariah.eu/beta/",
```
Comment (andmor-) [inline tools/DARIAH-DE-Geo-Browser-KML.json:27]: Same here

**Comment by emanueldima** [comment]
@fungunga Is the beta url final?

**Comment by fungunga** [comment]
No, Geo-Browser should have been released last week, but it didn't work out. I'll change that in a minute, we can use the current productive version, the URL will stay. I just wanted to test with the new BETA... thanks for the hint!

**Comment by fungunga** [comment]
...please let's try on test again...

**Comment by emanueldima** [comment]
Ok, can you make the PR in the master branch? Then I will merge the changes into this one.

File: tools/DARIAH-DE-Geo-Browser-CSV.json (hunk 1 above)
Comment (emanueldima) [reply tools/DARIAH-DE-Geo-Browser-CSV.json]: Thanks for the check André, these should be now solved.

**Review by andmor- — APPROVED** [review]

**Comment by andmor-** [comment]
Propagated to production. But `dariah-de.svg` does not exist in the `production` branch. I will take care of that

**Pull request merged**